internal/ingress/annotations/annotations.go

```diff
@@ -80,30 +80,31 @@ type Ingress struct {
 	CorsConfig           cors.Config
 	CustomHTTPErrors     []int
 	DefaultBackend       *apiv1.Service
-	Denied               error
-	ExternalAuth         authreq.Config
-	HTTP2PushPreload     bool
-	Proxy                proxy.Config
-	RateLimit            ratelimit.Config
-	Redirect             redirect.Config
-	Rewrite              rewrite.Config
-	Satisfy              string
-	SecureUpstream       secureupstream.Config
-	ServerSnippet        string
-	ServiceUpstream      bool
-	SessionAffinity      sessionaffinity.Config
-	SSLPassthrough       bool
-	UsePortInRedirects   bool
-	UpstreamHashBy       upstreamhashby.Config
-	LoadBalancing        string
-	UpstreamVhost        string
-	Whitelist            ipwhitelist.SourceRange
-	XForwardedPrefix     bool
-	SSLCiphers           string
-	Logs                 log.Config
-	LuaRestyWAF          luarestywaf.Config
-	InfluxDB             influxdb.Config
-	ModSecurity          modsecurity.Config
+	//TODO: Change this back into an error when https://github.com/imdario/mergo/issues/100 is resolved
+	Denied             string
```

Comment:
Why not `*string` so you can narrow the changes to just the type?

Reply:
That's a good idea. I'll make the change.